ADMVEDU/ViewControllers/FilterViewController.swift

```diff
@@ -29,46 +29,61 @@ class FilterViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
+        configureStackView()
+        configureNavigationController()
+        setOptions()
+    }
+
+    func configureStackView() {
+        view.addSubview(stackView)
+        stackView.axis = .vertical
+        stackView.distribution = .fillEqually
+        stackView.spacing = 20
+        setStackViewConstraints()
         configurePeakers()
         configureSwitch()
         configureSlider()
-        configureNavController()
-        setOptions()
+
     }
-    
+
+    func setStackViewConstraints() {
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor,
+                                       constant: 120).isActive = true
+        stackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+                                           constant: 40).isActive = true
+        stackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                                            constant: -40).isActive = true
+        stackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor,
+                                          constant: -350).isActive = true
+    }
+
```

Comment:
All numbers to constants. And may be it's time to add extension for UIView with methods for setting view's anchors (if planning to continue build UI with code)

Reply:
not sure if it's good idea. Maybe general style, but again constraint in the whole project in diffrent situation. @obazyleva  what do you think?

Reply:
should be moved to common separate method with params as childView and constants